test/cluster-py/multi.test.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import os
+from threading import Thread
+import time
+import yaml
+from lib.tarantool_server import TarantoolServer
+sys.path.append('../tarantool')
+from mesh_connection import MeshConnection
+from tarantool.const import (
+    SOCKET_TIMEOUT,
+    RECONNECT_DELAY,
+)
+from tarantool.error import NetworkError
+from tarantool.utils import ENCODING_DEFAULT
+
+REPLICA_N = 2
+
+
+def check_cluster(addrs):
+    con = MeshConnection(addrs=addrs,
+                         user=None,
+                         password=None,
+                         socket_timeout=SOCKET_TIMEOUT,
+                         reconnect_max_attempts=1,
+                         reconnect_delay=RECONNECT_DELAY,
+                         connect_now=True,
+                         encoding=ENCODING_DEFAULT)
+    while True:
+        try:
+            s = con.space('test')
+            print s.select()
+        except NetworkError:
+            print 'NetworkError !'
+            break
+        except Exception as e:
+            print e
+        time.sleep(1)
+
+
+# Start replicas
+master = server
+cluster = [master]
+for i in range(REPLICA_N):
+    server = TarantoolServer(server.ini)
+    server.script = 'cluster-py/replica.lua'
+    server.vardir = os.path.join(server.vardir, 'replica', str(i))
+    server.deploy()
+    server.admin("box.schema.user.grant('guest', 'read,write,execute', 'universe')")
+    server.admin("_ = box.schema.space.create('test')")
+    server.admin("_ = box.space.test:create_index('primary')")
+    server.admin("box.space.test:insert{%d, %s}" % (1, i), silent = True)
+    cluster.append(server)
+
+# Make a list of servers
+sources = []
+for server in cluster[1:]:
+    sources.append(yaml.load(server.admin('box.cfg.listen', silent=True))[0])
+
+addrs = []
+for addr in sources:
+    host, port = addr.split(':')
+    addrs.append({'host': host, 'port': int(port)})
+
+thread = Thread(target=check_cluster, args=(addrs, ))
```

Comment:
I think it is better to create purely sequencial test and don't lean on timings. Like so (schematically):

```
setup instances
setup mesh connection
perform select
stop instance 1
perform select
stop instance 2
start instance 1
perform select
stop instance 1
perform select
```